### ISSUE TITLE
Error when trying to download update to 2.12.6 on Ubuntu 16.04

### ISSUE THREAD

**Issue by rdebeasi** [avocode/avocode#1286]
I got this error when trying to download the latest version of the application. I'm running elementary OS 0.4 (Ubuntu 16.04).

I'm not logged into the Avocode website. Restarting the application and clicking the download button again didn't reproduce the problem.

I know that's super vague, but I hope it helps a little! :)

-----
App v. `2.11.1`
Avocode UID: 92162
```
TypeError: _jp.afvgxbs is not a function
    at https://manager.avocode.com/sockjs/sock/052/xtdkucgx/jsonp?c=_jp.afvgxbs:1:5
```

**Comment by connor-bean** [comment]
Hey @rdebeasi -

Thanks for the report and the recreation steps. So just to confirm - you're now running 2.12.6 and it's ok?

Thanks,
Connor, Avocode Support

**Comment by vojtatranta** [comment]
@rdebeasi this error was fixed in `>2.12.0` versions, so you should be okay.
